### PR TITLE
CFV page

### LINKED ISSUE #103: Create call for volounteers
Something similar to cfp i guess?!?!

### PR DESCRIPTION
**Pull request by Wabri**
Closes #103

### CODE REVIEW

**Review by TheJoin95 — APPROVED** [review]
Crazy Wab

**Comment by TheJoin95** [comment]
Good to go for me

**Review by 404answernotfound — APPROVED** [review]

File: components/CfvCard.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { TTypeCard } from '../constants';
+
+const CfvCard = ({ title, typeClass }: TTypeCard) => {
```
Comment (nicpuppa) [inline components/CfvCard.tsx:4]: This component is not used, wright ?

**Review by 404answernotfound — APPROVED** [review]

**Review by davideimola — APPROVED** [review]
LGTM, apart from the things @nicpuppa have already mentioned

File: components/CfvCard.tsx (hunk 1 above)
Comment (Wabri) [reply components/CfvCard.tsx]: You are the master of FE :1st_place_medal:

**Review by nicpuppa — APPROVED** [review]

**Pull request merged**